src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -645,7 +645,13 @@ class misc_commandscript : public CommandScript
 
     static bool HandleDismountCommand(ChatHandler* handler, char const* /*args*/)
     {
-        Player* player = handler->GetSession()->GetPlayer();
+        Player* player = handler->getSelectedPlayerOrSelf();
+        if (!player)
```

Comment:
This check is unnecessary now, but it will prevent server to crash if someone allow to use this command from console